File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.cs
using System;

public class Solution {
    public int solution(string binomial) {
        string [] arr = binomial.Split(' ');
        if(arr[1] == "+"){
            return int.Parse(arr[0]) + int.Parse(arr[2]);
        }
        else if(arr[1] == "-"){
            return int.Parse(arr[0]) - int.Parse(arr[2]);
        }
        else{
            return int.Parse(arr[0]) * int.Parse(arr[2]);
        }

    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    arr = binomial.split(" ")
    if(arr[1] == '+'):
        return int(arr[0]) + int(arr[2])
    elif(arr[1] == '-'):
        return int(arr[0]) - int(arr[2])
    elif(arr[1] == '*'):
        return int(arr[0]) * int(arr[2])
    return 1
